ga4gh/datamodel/variants.py

```diff
@@ -334,6 +334,195 @@ def generateVariant(self, referenceName, position, randomNumberGenerator):
         return variant
 
 
+class SimulatedVariantAnnotationSet(AbstractVariantSet):
+    """
+    A variant annotation set that doesn't derive from a data store.
+    Used mostly for testing.
+    """
+    def __init__(
+            self, parentContainer, localId, variantSet, randomSeed=1):
+        super(SimulatedVariantAnnotationSet, self).__init__(
+            parentContainer, localId)
+        self._variantSet = variantSet
+        self._variantSetId = str(variantSet.getCompoundId())
+        self._randomSeed = randomSeed
+        now = protocol.convertDatetime(datetime.datetime.now())
+        self._compoundId = datamodel.VariantAnnotationSetCompoundId(
+            self.getCompoundId(), 'variantannotations')
+        self._creationTime = now
+        self._updatedTime = now
+        # TODO make some reasonable connection to seqont
+
+    def getAnalysis(self):
+        analysis = protocol.Analysis()
+        analysis.recordCreateTime = datetime.datetime.now().strftime(
+                                    "%Y-%m-%dT%H:%M:%S.%fZ")
+        analysis.recordUpdateTime = datetime.datetime.now().strftime(
+                                    "%Y-%m-%dT%H:%M:%S.%fZ")
+        analysis.software.append("software")
+        analysis.name = "name"
+        analysis.description = "description"
+        analysis.id = str(datamodel.VariantAnnotationSetAnalysisCompoundId(
+                    self._compoundId, "analysis"))
```

Comment:
This is new enough all right --- do you have a git commit-hook installed to run flake8? The situation here is that we're using an old version of flake8 in Travis for historical reasons, but it's not worth updating it now because of all the branches that are in flight. It seems like flake8 has tightened up some checks quite a bit recently, which causes issues for people who do have a flake8 git commit hook with a newer version of flake8.

Anyway, I should just downgrade my flake8 to the version in requirements.txt and stop worrying about it!